QuickFIXn/SocketReader.cs

```diff
@@ -19,7 +20,9 @@ public class SocketReader : IDisposable
     private readonly TcpClient _tcpClient;
     private readonly ClientHandlerThread _responder;
     private readonly AcceptorSocketDescriptor? _acceptorDescriptor;
-    private readonly NonSessionLog _nonSessionLog;
+    private readonly ILogger _nonSessionLog;
+
+    private ILogger UnconditionalLogger => _qfSession?.Log is not null ? _qfSession.Log : _nonSessionLog;
```

Comment:
another syntax choice 🙂 

```suggestion
    private ILogger UnconditionalLogger => _qfSession?.Log ?? _nonSessionLog;
```

Reply:
of course, duhhhhhhhhhh on me!  Will be in the next commit.